src/messages.h

```diff
@@ -13,6 +13,7 @@ class JsonOut;
 class JsonObject;
 
 enum game_message_type : int;
+extern bool log_from_top;
```

Comment:
It shouldn't be in a header when it's only used in one cpp file, but just directly in the cpp file itself.